stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/notifications/DefaultNotificationHandler.kt

```diff
@@ -120,12 +127,15 @@ public open class DefaultNotificationHandler(
                 logger.e { "Couldn't find any activity for $ACTION_MISSED_CALL" }
                 intentResolver.getDefaultPendingIntent()
             }
-        StreamVideo.instance().state.ringingCall
-        showMissedCallNotification(
-            intent,
-            callDisplayName,
-            notificationId,
-        )
+        val notification = getMissedCallNotification(callId, callDisplayName)
```

Comment:
The code just above this line seems to be useless now, I think it can be deleted